frontend/src/components/script-editor.tsx

```diff
@@ -71,14 +74,18 @@ export default function ScriptEditor() {
         )}
       >
         <Editor
-          beforeMount={setEditorTheme}
+          beforeMount={(monaco) => {
+            setEditorTheme(monaco);
+            monaco.editor.remeasureFonts();
```

Comment:
Can we move `monaco.editor.remeasureFonts()` into the `setEditorTheme` function?